demos/demo-react-ts/src/hooks/useCti.ts

```diff
@@ -154,17 +154,19 @@ export const useCti = (
       debugMode: true,
       eventHandlers: {
         onReady: (data: { engagementId: number | undefined }) => {
+          const { engagementId } = data || {};
           cti.initialized({
             isLoggedIn: true,
             sizeInfo: defaultSize,
-            engagementId: data.engagementId,
+            engagementId,
```

Comment:
@esme, feels weird to not update this demo with the `portalId` and `userId`. Should I send it in the `initialized` event? This may make the developer think they need to send it...

Reply:
I don't think it is necessary to include the `portalId` and `userId` in the initialized payload since HubSpot already has the information but I also don't see any downsides to doing so. Feel free to make the judgment call here!

Reply:
I'll leave it off!